### PR TITLE
Relax version bounds of warp, text and filepath

### PR DESCRIPTION
**Pull request by jhrcek**
https://github.com/commercialhaskell/stackage/issues/7287

### CODE REVIEW

File: elm-street.cabal [hunk 1]
```diff
@@ -75,9 +75,9 @@ library
 
   build-depends:       aeson >= 1.3
                      , directory ^>= 1.3
-                     , filepath ^>= 1.4
+                     , filepath >= 1.4 && < 1.6
                      , prettyprinter >= 1.2.1 && < 1.8
-                     , text >= 1.2 && <= 2.1
+                     , text >= 1.2 && <= 2.2
```
Comment (turboMaCk) [inline elm-street.cabal:80]: is strict upper bound really that useful here?

Comment (jhrcek) [reply elm-street.cabal:80]: What do you propose?
Using some arbitrary other bound?
Dropping the upper bound entirely?

Comment (turboMaCk) [reply elm-street.cabal:80]: I'm not really sure. Maybe we can stretch this to < 3.0 ?

Comment (jhrcek) [inline elm-street.cabal:80]:
```suggestion
                     , text >= 1.2 && <= 3.0
```

**Review by turboMaCk — APPROVED** [review]
LGTM

**Pull request merged**